ProcessMaker/Http/Controllers/Auth/LoginController.php

```diff
@@ -55,6 +56,25 @@ public function showLoginForm()
     {
         $manager = App::make(LoginManager::class);
         $addons = $manager->list();
+        $default = Setting::where('key', 'sso.default.login')->first();
```

Comment:
Is it possible to obtain $element->name from a database query?

Reply:
As i show you this element handles a fairly extensive json in which the name is found within it, so it is not possible